src/design/icons/icons.stories.mdx

```diff
@@ -41,7 +41,7 @@ For icon usage examples, see [the icon object](/?path=/docs/components-icon--bas
 
 <IconGallery>{iconElements}</IconGallery>
 
-# Brand Icons
+# Third Party Icons
```

Comment:
I believe "Third-Party" should be hyphenated since it is an adjective?

Reply:
Good point. Regarding case, my instinct is to use "Third-party Icons" instead of of "Third-Party Icons". 

Let me know if you have a preference.